modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -628,104 +684,94 @@ class CV_EXPORTS_W VideoCapture
 
     Also, when a connected camera is multi-head (for example, a stereo camera or a Kinect device), the
     correct way of retrieving data from it is to call VideoCapture::grab first and then call
-    VideoCapture::retrieve one or more times with different values of the channel parameter. See
-    <https://github.com/opencv/opencv/tree/master/samples/cpp/openni_capture.cpp>
+    VideoCapture::retrieve() one or more times with different values of the channel parameter.
+
+    See <https://github.com/opencv/opencv/tree/master/samples/cpp/openni_capture.cpp>
```

Comment:
It will be better to provide a link to the Kinect/OpenNI tutorial: `@ref tutorial_kinect_openni`